cukinia

```diff
@@ -524,4 +611,14 @@ cukinia_tests=0
 cukinia_failures=0
 cukinia_conf_include "$CUKINIA_CONF"
 
+# Create junitxml report
+if [ "$__log_format" = "junitxml" ]; then
+	_junitxml_start >"$__log_file".xml
+	cat "$__logfile_tmp" >> "$__log_file".xml
+	_junitxml_end >>"$__log_file".xml
+fi
+
+# Cleanup of temporary files
+rm "$__stderr_tmp" "$__stdout_tmp" "$__logfile_tmp"
```

Comment:
you can rely on a cleanup function for that if you trap exits

Reply:
ack